docs/bitcoin/explorer.md

```diff
@@ -1 +1,17 @@
 # Explorer
+
+## Overview
+Blockchain explorers are GUI applications built over the Bitcoin blockchain to see transactions and public addresses. Generally, mobile and hardware wallets use trusted third-party explorers to check wallet balances and confirm transactions which leaks the private details (like IP address, bitcoin balance). Instead, having a local explorer with personal Bitcoin node will improve privacy and remove the trust on third-party explorers.
```

Comment:
Blockchain explorers are GUI applications built over the Bitcoin blockchain to see transactions and public addresses. Generally, mobile and hardware wallets use trusted third-party explorers to check wallet balances and confirm transactions, which leak the private details (like IP address, bitcoin balance). Instead, having a local explorer with a personal Bitcoin node will improve privacy and remove the trust on third-party explorers.

Reply:
Sorry, I don't see your minor changes.

Reply:
My bad...I totally messed up the review process here and elsewhere. Hopefully when I'm done here you'll see my suggested changes.